tests/harness/run_tests.sh

```
@@ -0,0 +1,123 @@
+      # Expect at least two tokens: STATUS and FILE
+      status=$(printf '%s' "$line" | awk '{print $1}')
+      file=$(printf '%s' "$line" | awk '{print $2}')
+      if [[ -z "$status" || -z "$file" ]]; then continue; fi
+      echo "  <testcase classname=\"compliance\" name=\"$file\">"
+      if [[ "$status" == "FAIL" ]]; then
+        echo "    <failure message=\"failed\"/>"
+      elif [[ "$status" == "SKIP" ]]; then
+        echo "    <skipped/>"
+      fi
+      echo "  </testcase>"
+    done <<< "$(printf \"%b\" \"$results_lines\")"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)